### ISSUE TITLE
`testShaftRotatesInAutonomous` fails occasionally

### ISSUE THREAD

**Issue by brettle** [DeepBlueRobotics/DeepBlueSim#94]
It just succeeded on my local machine 17 times in a row and then failed with:

```
org.opentest4j.AssertionFailedError: Shaft close to target rotation ==> expected: <160.0> but was: <95.42188982042626>
	at app//org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at app//org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:62)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:86)
	at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1020)
	at app//frc.robot.PWMMotorControllerTest.lambda$testShaftRotatesInAutonomous$1(PWMMotorControllerTest.java:31)
	at app//org.carlmontrobotics.libdeepbluesim.WebotsSimulator.runAllCallbacks(WebotsSimulator.java:467)
	at app//org.carlmontrobotics.libdeepbluesim.WebotsSimulator.lambda$waitForUserToStart$6(WebotsSimulator.java:377)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base@17.0.8.1/java.lang.Thread.run(Thread.java:833)
```

Might just need to increase the tolerance.

**Comment by brettle** [comment]
This is probably best worked on after #88.

**Comment by brettle** [comment]
Will probably be fixed by a fix to #101.

**Comment by brettle** [comment]
Should be addressed by PR #114. Reopen if another assertion failure is seen.
